tests/zfs-tests/tests/functional/fault/scrub_after_resilver.ksh

```diff
@@ -0,0 +1,66 @@
+#!/bin/ksh -p
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2018 by Lawrence Livermore National Security, LLC.
+# All rights reserved.
+#
+
+. $STF_SUITE/include/libtest.shlib
+. $STF_SUITE/tests/functional/fault/fault.cfg
+
+#
+# DESCRIPTION:
+# Test the scrub after resilver zedlet
+#
+# STRATEGY:
+# 1. Create a mirrored pool
+# 2. Fault a disk
+# 3. Replace the disk, starting a resilver
+# 4. Verify that a scrub happens after the resilver finishes
+#
+
+log_assert "Testing the scrub after resilver zedlet"
+
+# Backup our zed.rc
+old_zedrc="$(cat $ZEDLET_DIR/zed.rc)"
+
+# Enable ZED_SCRUB_AFTER_RESILVER
+log_must sed -i 's/\#ZED_SCRUB_AFTER_RESILVER/ZED_SCRUB_AFTER_RESILVER/g' $ZEDLET_DIR/zed.rc
+
+function cleanup
+{
+	# Restore our zed.rc
+	log_must echo "$old_zedrc" > $ZEDLET_DIR/zed.rc
+
+	default_cleanup
+}
+
+log_onexit cleanup
+
+verify_disk_count "$DISKS" 3
+default_mirror_setup_noexit $DISKS
+
+log_must zpool offline -f $TESTPOOL $DISK1
+log_must zpool detach $TESTPOOL $DISK2
```

Comment:
Why include `DISK2` in the pool at all which forces you to have to detach it?  By default only a two way mirror is setup and the 3rd disk should be available for the replace.  Although you may need to wipe it with `zpool labelclear`.